src/styles/blocks/card.scss

```diff
@@ -0,0 +1,85 @@
+.card {
+  max-width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+
+  &__img {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    color: $mainColor;
+    line-height: 150%;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    color: $greyColor;
+    margin-bottom: 16px;
+    line-height: 140%;
+  }
+
+  &__box {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 24px;
+
+    &--retreat {
+      margin-bottom: 16px;
+    }
+  }
+
+  &__revie {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 140%;
+  }
+
+  &__price-text {
+    font-size: 12px;
+    font-weight: 400;
+    color: $greyColor;
+    line-height: 150%;
+  }
+
+  &__price {
+    font-weight: 700;
+    color: $mainColor;
+    line-height: 112.5%;
+  }
+
+  &__btn {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-decoration: none;
+    width: 100%;
+    height: 40px;
```

Comment:
it's better not hardcode height for elements with content inside. Just use padding: 11px 0;

Reply:
Which difference between these two options: 'height: 40px; and padding: 11px 0;'? I think using second option is hardcode too?